bindings.cpp

```diff
@@ -1413,12 +1417,12 @@ bool DeRestPluginPrivate::sendConfigureReportingRequest(BindingTask &bt)
             return sendConfigureReportingRequest(bt, {rq, rq2, rq3}) || // Use OR because of manuf. specific attributes
                    sendConfigureReportingRequest(bt, {rq4, rq5});
         }
-        else if (sensor && (sensor->modelId() == QLatin1String("0x8020") || // Danfoss RT24V Display thermostat
-                            sensor->modelId() == QLatin1String("0x8021") || // Danfoss RT24V Display thermostat with floor sensor
-                            sensor->modelId() == QLatin1String("0x8030") || // Danfoss RTbattery Display thermostat
-                            sensor->modelId() == QLatin1String("0x8031") || // Danfoss RTbattery Display thermostat with infrared
-                            sensor->modelId() == QLatin1String("0x8034") || // Danfoss RTbattery Dial thermostat
-                            sensor->modelId() == QLatin1String("0x8035")))  // Danfoss RTbattery Dial thermostat with infrared
+        else if (sensor && (modelId == QLatin1String("0x8020") || // Danfoss RT24V Display thermostat
```

Comment:
I'd assume we could also omit the `sensor &&` here?

Reply:
Good catch, since the `modelId` is so short perhaps it might be good to include the `manufacturerCode` in the check? Might be useful in general but I was afraid to break anything.